### PR TITLE
Fix ProfileEditing & navigation, clean and refactor

### PR DESCRIPTION
**Pull request by mblasiak**
A bit big but worth it

### CODE REVIEW

**Comment by mblasiak** [comment]
I removed redundant files/containers/classe, extracted some styles,color and reformated code.
We might want to use editroConfig

File: src/components/context/AppContext.js [hunk 1]
```diff
@@ -1,65 +1,74 @@
...
-  dispatch({ type: 'logOut' });
+    dispatch({type: 'logOut'});
 }
+
+async function updateUser(dispacth, user) {
+    //TODO: HIT BACKEDN WITH UPDATED USER
+    await saveDataInStorage('USER', JSON.stringify(user));
+    dispacth({type: "updateUser", payload: user});
+}
+
```
Comment (pectom) [inline src/components/context/AppContext.js]: typo with dispatch

File: src/components/ksButton/KsButton.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+import {StyleSheet, Text, TouchableWithoutFeedback, View} from "react-native";
+import {ksBasic} from "../../styles/basic/ksBasic";
```
Comment (pectom) [inline src/components/ksButton/KsButton.js]: Why ks?

Comment (pectom) [reply src/components/ksButton/KsButton.js]: Probably Krakostop. I just got it :+1:

Comment (gtluszcz) [reply src/components/ksButton/KsButton.js]: hahah love it :)

**Comment by gtluszcz** [comment]
It's okay to move some styles to configs, font-sizes, colors etc, but for more complicated stuff we could do our own components instead of advanced stylesheet objects

File: src/views/loginScreen/LoginComponent.js [hunk 3]
```diff
@@ -1,73 +1,62 @@
...
-        <View style={styles.loginBtnContainer}>
-          <TouchableWithoutFeedback onPress={_loginPressed}>
-            <Image style={styles.loginButton} source={loginButton} />
-          </TouchableWithoutFeedback>
+    return (
+        <View style={ksBasic.stackContainer}>
+            <View style={ksBasic.stack}>
+                <Image style={styles.logo} source={logo}/>
+                <View style={styles.loginBtnContainer}>
+                    <TouchableWithoutFeedback onPress={_loginPressed}>
```
Comment (gtluszcz) [inline src/views/loginScreen/LoginComponent.js]: we could move all Touchable and everything he has inside to separate component

Comment (gtluszcz) [reply src/views/loginScreen/LoginComponent.js]: but on the other hand we use it only here so i guess no point

**Comment by gtluszcz** [comment]
oh shit I'm dumb you actually did our own buttons etc

**Pull request merged**